docs/how-to/deploy.md

```diff
@@ -0,0 +1,46 @@
+---
+id: deploy
+title: Amplication Deployment
+sidebar_label: Overview
+slug: /deploy
+---
+
+# Amplication Deployment
+
+Amplication provides flexible and powerful deployment options for services generated using our platform.
+This guide provides an overview of Amplication's deployment options and links to specific instructions for various deployment targets.
+
+## Understanding Amplication's Deployment Architecture
+
+Applications created with Amplication comprise two main components:
+
+1. The server that runs the API
+2. A database
+
+When it comes to deployment, you have several options to choose from, allowing you to select the best approach for your development workflow and production needs.
+
+## Deployment Options
+
+### Docker
+
+Amplication generates all necessary configuration and scripts for Docker deployment, making it easy to containerize your application.
+
+[Learn more about deploying with Docker](/deploy/docker-desktop)
+
+### Kubernetes
+
+For those looking to leverage the power of Kubernetes orchestration, Amplication provides the necessary components for a smooth deployment process.
+
+[Learn more about deploying to Kubernetes](/deploy/kubernetes)
+
+### Amazon Web Services (AWS) ECS
+
+Amplication supports deployment to Amazon Web Services using Elastic Container Service (ECS). This option allows you to leverage AWS's managed container orchestration service for deploying, managing, and scaling your containerized applications.
+
+[Learn more about deploying to AWS ECS](/deploy/aws/ecs)
+
+## Request New Deployment Guides
+
+We're always looking to expand our deployment options to meet the needs of our users. Is there a specific deployment target or cloud provider you'd like to see covered in our documentation? We'd love to hear from you!
+
+[Contact us with your deployment guide requests](https://amplication.com/contact-us)
```

Comment:
I think the content of the link is not so relevant.
Let's add a link to my calendar to schedule a call to discuss the request or such

Reply:
Sounds good, @PazYanoverr. I'll update this. Can I get the link to your calendar?

Reply:
https://meetings-eu1.hubspot.com/paz-yanover/product-overview-vp-product